### DISCUSSION TITLE
Change PF style

### DISCUSSION THREAD

**Discussion by SMBEN** [mtex-toolbox/mtex#1418] [Ask Anything] [answered]
Hi, 
I want to change my PF style to be like the second picture
![2P Mg](https://user-images.githubusercontent.com/99767925/166128744-c88cb73d-0c98-4e94-b8d6-abc6c2560602.jpg)
![MMe](https://user-images.githubusercontent.com/99767925/166128758-b241199f-bb20-4027-9386-2b526bba7e25.png)

**Comment by kilir** [comment, marked as answer]
Hi,
you could start here:
https://mtex-toolbox.github.io/Annotations.html
https://mtex-toolbox.github.io/PlotTypes.html
Cheers,
Rüdiger

**Reply by SMBEN** [reply]
Hi kilir, 
thank you for your guidance.
